src/runtime/classmanager.cs

```diff
@@ -82,30 +82,31 @@ private static int TraverseTypeClear(IntPtr ob, IntPtr arg)
 
         internal static void SaveRuntimeData(RuntimeDataStorage storage)
         {
-            var contexts = storage.AddValue("contexts",
-                new Dictionary<IntPtr, InterDomainContext>());
-            storage.AddValue("cache", cache);
-            foreach (var cls in cache.Values)
-            {
-                // This incref is for cache to hold the cls,
-                // thus no need for decreasing it at RestoreRuntimeData.
-                Runtime.XIncref(cls.pyHandle);
-                var context = contexts[cls.pyHandle] = new InterDomainContext();
-                cls.Save(context);
-            }
+            // var contexts = storage.AddValue("contexts",
```

Comment:
The only thing saved here is the cache, and we don't want to. The cache is actually cleared in Runtime.Shutdown()